src/encoder.rs

```diff
@@ -3156,7 +3156,7 @@ fn encode_partition_topdown<T: Pixel, W: Writer>(
             &Some(PartitionGroupParameters {
               rd_cost: mode.rd_cost,
               part_type: PartitionType::PARTITION_NONE,
-              part_modes: once(mode).collect(),
+              part_modes: ArrayVec::try_from(&[mode] as &[_]).unwrap(),
```

Comment:
Is the following equivalent?
```suggestion
              part_modes: [mode][..].try_into().unwrap(),
```

Reply:
It should be, since `try_into` calls `try_from` anyway.